MorphicServer/RegisterEndpoint.cs

```diff
@@ -97,6 +100,28 @@ public async Task Post()
             }
         }
 
+        static private readonly int MinPasswordLength = 8;
+        public static readonly ReadOnlyCollection<string> BadPassword = new ReadOnlyCollection<string>(
```

Comment:
This is just a quick and dirty list. This will need to get expanded later in some way.